app/tests/integration/nimbus/test_experimenter.py

```diff
@@ -52,3 +54,35 @@ def test_promote_to_rollout(
     summary_detail = summary.navigate_to_details()
     summary_detail.promote_first_branch_to_rollout()
     summary_detail.wait_for_clone_parent_link_visible()
+
+
+@pytest.mark.run_once
+def test_takeaways(
+    selenium,
+    kinto_url,
+    kinto_review_url,
+    experiment_url,
+    create_experiment,
+):
+    create_experiment(selenium).launch_and_approve()
+
+    Login(selenium, kinto_url).open().login()
+    Dashboard(selenium, kinto_review_url).open().approve()
+
+    summary = SummaryPage(selenium, experiment_url).open()
+    summary.wait_for_live_status()
+    summary.end_and_approve()
+
+    Dashboard(selenium, kinto_review_url).open().approve()
+
+    summary = SummaryPage(selenium, experiment_url).open()
+    summary.wait_for_complete_status()
```

Comment:
Kind of a shame we have to go through this whole process just to get to complete status.  I've been musing on how we can use the factories through a command line to setup some experiments in specific states, but we can do that later.  For now this looks exactly right 👍